mlir/lib/Gradient/Transforms/BufferizationPatterns.cpp

```diff
@@ -74,21 +72,22 @@ class BufferizeBackpropOp : public OpConversionPattern<BackpropOp> {
 
         int argsSize = args.size();
 
-        int m = argsSize / resSize;
+        int divResSizeArgSize = resSize / argsSize;
```

Comment:
Maybe `numCalleeResults` or something conveys more information than how the value is calculated 😅